examples/timestamping/backend/tests/config.rs

```diff
@@ -75,21 +73,24 @@ fn init_testkit(second_time_service: bool) -> (TestKit, MockTimeProvider) {
     (testkit, mock_provider)
 }
 
-/// Creates block with `ConfigPropose` tx and returns current service configuration.
-fn propose_configuration(testkit: &mut TestKit, config: Config) -> Config {
+/// Creates block with `ConfigPropose` tx and returns `Result` with new configuration or corresponding `ExecutionError`.
+fn propose_configuration(testkit: &mut TestKit, config: Config) -> Result<Config, ExecutionError> {
     let tx = ConfigPropose::immediate(0).service_config(SERVICE_ID, config);
 
-    let initiator_id = testkit.network().us().validator_id().unwrap();
-    let (pub_key, sec_key) = &testkit.validator(initiator_id).service_keypair();
-    testkit.create_block_with_transaction(tx.sign_for_supervisor(*pub_key, sec_key));
-
-    testkit
-        .snapshot()
-        .for_service(SERVICE_NAME)
-        .unwrap()
-        .get_entry("config")
-        .get()
-        .unwrap()
+    let (pub_key, sec_key) = testkit.network().us().service_keypair();
+    let tx = tx.sign_for_supervisor(pub_key, &sec_key);
+    let block = testkit.create_block_with_transaction(tx.clone());
+
+    if let Err(e) = block.index(tx.object_hash()).status() {
```

Comment:
Nit: `block[tx.object_hash()]` should work (or `block[0]` for that matter; `create_block_with_transaction` creates a block with exactly the specified transaction and nothing else). `block[0]` would also allow to avoid cloning `tx`.